### PR TITLE
Add Glosbe API

### PR DESCRIPTION
**Pull request by arvindh-manian**
Thank you for taking the time to work on a Pull Request for this project!

To ensure your PR is dealt with swiftly please check the following:

- [√ ] Your submissions are formatted according to the guidelines in the [contributing guide](CONTRIBUTING.md)
- [ √] Your additions are ordered alphabetically
- [ √] Your submission has a useful description
- [ √] The description does not end with punctuation
- [ √] Each table column should be padded with one space on either side
- [ √] You have searched the repository for any relevant issues or pull requests
- [ √] Any category you are creating has the minimum requirement of 3 items
- [ √] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -313,6 +314,7 @@ API | Description | Auth | HTTPS | CORS |
 |---|---|---|---|---|
 | [Cloudmersive Document and Data Conversion](https://cloudmersive.com/convert-api) | HTML/URL to PDF/PNG, Office documents to PDF, image conversion | `apiKey` | Yes | Yes |
 | [File.io](https://www.file.io) | File Sharing | No | Yes | Unknown |
+| [html2pdf](https://html2pdf.app/) | HTML/URL to PDF | `apiKey` | Yes | Unknown |
```
Comment (SuperSandro2000) [inline README.md]: This does not belong in this PR.

Comment (arvindh-manian) [reply README.md]: Fixed

**Comment by marekdano** [comment]
@arvindh-manian thanks for your contribution! 
The link `https://glosbe.com/a-api` looks to be broken. Can you please review it?

**Comment by matheusfelipeog** [comment]
This API has been deprecated:

https://glosbe.com/help/api
image
